core/gen_genesis.go

```diff
@@ -7,19 +7,19 @@ import (
 	"errors"
 	"math/big"
 
-	"github.com/ava-labs/subnet-evm/core/types"
 	"github.com/ava-labs/libevm/common"
 	"github.com/ava-labs/libevm/common/hexutil"
 	"github.com/ava-labs/libevm/common/math"
-	"github.com/ava-labs/libevm/params"
+	"github.com/ava-labs/libevm/core/types"
+	params0 "github.com/ava-labs/subnet-evm/params"
```

Comment:
nit: if we must add an alias here, can we rename this?

Reply:
This is generated code